test/src/utils/online_statistics.test.cpp

```diff
@@ -156,19 +157,23 @@ TEST_F(onlineStatistics_test_int, test_int_outliers)
   for (int i = 0;i < (static_cast<int>(values_counter)/10);i++) {
     test_stats_int.update(threshold +  rand() % 200 + 100);
   }
-  EXPECT_LT(mean_prev, test_stats_int.getMean());
-  EXPECT_LT(var_prev, test_stats_int.getVariance());
-  EXPECT_LT(std_dev_prev, test_stats_int.getStdDev());
+  EXPECT_LT(mean_prev, test_stats_int.getMean())
+      << "Outliers do not affect the mean as expected.";
```

Comment:
maybe is a good idea to encapsulate the 'error' messages in variables, it is cleaner and also avoids repetition like in this case :)